notices.json

```diff
@@ -17,21 +17,21 @@
     }
   },
   {
-    "id": "server_upgrade_v5.29",
+    "id": "server_upgrade_v5.30",
     "conditions": {
       "audience": "sysadmin",
       "clientType": "all",
-      "serverVersion": ["<5.29"],
+      "serverVersion": ["<5.30"],
       "instanceType": "onprem",
-      "displayDate": ">= 2020-11-19T00:00:00Z"
+      "displayDate": ">= 2020-12-19T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "New server version available",
-        "description": "Mattermost server v5.29 is now available offering many bug fixes for increased stability as well as the general availability of a pre-installed incident management application (E20), channel moderation settings (E20), and Mattermost Omnibus. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
+        "description": "Mattermost server v5.30 is now available offering Incident Management out-of-the-box, a new MS Teams Meetings plugin, configuration of new administrator roles from the System Console, and improvements to the Matterpoll plugin. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.",
```

Comment:
Propose:
Mattermost server v5.30 is now available offering a new MS Teams Meetings plugin, new administrator roles configurable in the System Console, and improvements to the Matterpoll plugin. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) only takes a few minutes.

_______

> Incident Management out-of-the-box, 
Not really clear how this is different than the text in v5.29 "pre-installed incident management application (E20)"? Propose we just remove it?